operations/app/src/modules/storage/candidate_slot.tf

```diff
@@ -68,6 +70,40 @@ module "storageaccount_candidate_queue_private_endpoint" {
   resource_group     = var.resource_group
   location           = var.location
   endpoint_subnet_id = data.azurerm_subnet.endpoint.id
+  create_dns_record  = true
+}
+
+module "storage_candidate_blob_private_endpoint" {
+  source             = "../common/private_endpoint"
+  resource_id        = azurerm_storage_account.storage_account_candidate.id
+  name               = azurerm_storage_account.storage_account_candidate.name
+  type               = "storage_account_blob"
+  resource_group     = var.resource_group
+  location           = var.location
+  endpoint_subnet_id = data.azurerm_subnet.endpoint_subnet.id
+  create_dns_record  = false
+}
+
+module "storage_candidate_file_private_endpoint" {
+  source             = "../common/private_endpoint"
+  resource_id        = azurerm_storage_account.storage_account_candidate.id
+  name               = azurerm_storage_account.storage_account_candidate.name
+  type               = "storage_account_file"
+  resource_group     = var.resource_group
+  location           = var.location
+  endpoint_subnet_id = data.azurerm_subnet.endpoint_subnet.id
+  create_dns_record  = false
+}
+
+module "storage_candidate_queue_private_endpoint" {
+  source             = "../common/private_endpoint"
+  resource_id        = azurerm_storage_account.storage_account_candidate.id
+  name               = azurerm_storage_account.storage_account_candidate.name
+  type               = "storage_account_queue"
+  resource_group     = var.resource_group
+  location           = var.location
+  endpoint_subnet_id = data.azurerm_subnet.endpoint_subnet.id
+  create_dns_record  = false
```

Comment:
These private endpoints were actually missing from the original Palo Alto network PR.